### DISCUSSION TITLE
笑的基本原理

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1549] [General]
我之前看了一本喜劇的書，內容在教導怎麼製作出一個喜劇的段子，並把為什麼會「笑」分成了四種情況。
分為以下四種：
反邏輯、身體的痛苦、羞恥的行為、醜化。

反邏輯：透過意料之外的邏輯造成笑點
如 黃耀明非常認真讀書，因此考進了台大醫學院......的精神病院
身體的痛苦：像看到ig 短片裡面有人跌倒或下面被打倒
羞恥的行為：公共場合出糗
醜化：小丑或是任何外貿人設很愚笨的人事物

**Comment by poweichen00** [comment]
這些情況的確是在喜劇中常見的元素，可以讓觀眾們笑，而喜劇是一種主觀的體驗，每個人對於什麼讓他們笑是不同的，這些情況僅僅是常見的喜劇元素之一，喜劇的創作和詮釋有很大的靈活性和多樣性。

**Comment by larry40305** [comment]
我認為高明的反邏輯能造成比其他種類的情況更有效的笑果，尤其是那種能前後呼應地段子更能讓人回味無窮。

**Comment by zhiron544** [comment]
我覺得這篇文章很有趣，讓我可以更清楚地分析不同類型的笑話。我同意反邏輯是一種很高級的喜劇手法，需要有很好的觀察力和創造力。我也想知道其他人對於身體的痛苦、羞恥的行為、醜化這三種情況有什麼看法，是否覺得它們也有趣或是有道德問題。

**Comment by 110302001** [comment]
我之前聽別人說笑是因為有覺得荒謬的感覺，我想應該就是屬於上面說的反邏輯的部分吧，另外我覺得讓人產生共鳴也會引人發笑，會有相同記憶之類的。
